python/lsst/sims/GalSimInterface/galSimDetector.py

```diff
@@ -90,6 +101,28 @@ def __init__(self, detectorName, cameraWrapper, obs_metadata, epoch, photParams=
         if self.obs_metadata.rotSkyPos is not None:
             self.fitsHeader.set('ROTANGLE', obs_metadata.rotSkyPos)
 
+        # Add airmass, needed by jointcal.
+        try:
+            airmass = self.obs_metadata.OpsimMetaData['airmass']
+        except KeyError:
+            pass
+        else:
+            self.fitsHeader.set('AIRMASS', airmass)
```

Comment:
Josh had requested 

```
• parallactic angle
• temperature
• pressure
• water-vapor partial pressure
to the list if they're available, as these all also affect DCR.
```

Are they available from the same OpSim information at this point?

Reply:
None of these are available.